test/nettle.jl

```diff
@@ -0,0 +1,32 @@
+import Nettle
+
+function test_equal_state(m::MD5.MD5_CTX,n::Nettle.Hasher)
+    @test (n.state)[1:16] == reinterpret(UInt8, m.state)
+end
+
+@testset "Against Nettle initialize, update!, digest!" begin
+    for chunkcount ∈ [0,1,2,3, 10, 100]
+        m = MD5.MD5_CTX()
+        n = Nettle.Hasher("md5")
```

Comment:
maybe a more descriptive name,
like `md5_ctx` and `nettle_ctx`

or even `md5_value` and `nettle_value`